src/helpers/convert-from-xml.ts

```diff
@@ -178,7 +178,7 @@ export const convertXmlToText = async (
 
     // Use thread authors if found, otherwise fall back to legacy parsing
     if (authorsForFeed.length === 0) {
-      console.log("No thread structure found, parsing legacy flat author structure");
+      // console.log("No thread structure found, parsing legacy flat author structure");
```

Comment:
A few `console.log()` statements left in this file. I am okay with this, but I wanted to point it out in case anyone else has a strong opinion.